guides/coding_style/c++/style.md

```diff
@@ -279,4 +281,38 @@ enum class ExampleEnum
     THREE,
     FOUR // No comma at the end
 };
+
+enum class ExampleEnum : long
+{
+    ONE = 0,
+    TWO_TWO,
+    THREE,
+    FOUR // No comma at the end
+};
+```
+
+Alias
+===
+
+using vs typedef
+---
+When dealing with aliases, use `using`, not `typedef`.
+
+`using` can be read almost as an english sentence.
+
+More easily define alias with template.
+
+[more comparison](http://www.stroustrup.com/C++11FAQ.html#template-alias)
+
+Naming
+---
+
+Aliases should be named like lowercase or lowercase_with_underscore.
```

Comment:
Recommend PascalCase here, since you are basically creating a _type_ through an alias.